setup.py

```diff
@@ -15,7 +15,7 @@
 REQUIRED_PKGS = [
     "coloredlogs",
     "sympy",
-    "transformers[sentencepiece]>=4.26.0,<4.43.0",
+    "transformers[sentencepiece]>=4.33.0,<4.43.0",
```

Comment:
we don't support 4.26.0 since a long time https://github.com/huggingface/optimum/blob/5ece6e88156de6a0de848ad6c1af381c81cfbcc3/optimum/exporters/onnx/model_patcher.py#L25 at least for `optimum.exporters` and looks like it won't impact any other subpackage : 
amd : https://github.com/huggingface/optimum-amd/blob/main/setup.py#L17
intel : https://github.com/huggingface/optimum-intel/blob/v1.18.1/setup.py
habana : https://github.com/huggingface/optimum-habana/blob/v1.12.1/setup.py#L32
neuron : https://github.com/huggingface/optimum-neuron/blob/v0.0.23/setup.py

Reply:
will impact furiosa that with "transformers>=4.20.0"  https://github.com/huggingface/optimum-furiosa/blob/main/setup.py#L16 cc @mht-sharma

Reply:
should be done in an other PR with tests, reverting